tests/kafkatest/tests/streams/streams_optimized_test.py

```diff
@@ -96,7 +96,7 @@ def verify_running_repartition_topic_count(processor, repartition_topic_count):
         with node.account.monitor_log(processor.STDOUT_FILE) as monitor:
             processor.start()
             monitor.wait_until('REBALANCING -> RUNNING with REPARTITION TOPIC COUNT=%s' % repartition_topic_count,
-                               timeout_sec=60,
+                               timeout_sec=120,
```

Comment:
Need to increase timeout for startup and rebalance to running, noticing some failures due longer times.  I'll investigate this further, but for now increasing the timeout to stop the bleeding.